packages/colors/package.json

```diff
@@ -8,9 +8,8 @@
         "compile": "run-p \"compile:*\"",
         "compile:esm": "tsc -p src/",
         "compile:css": "sass-compile ./src",
+        "compile:colors-css": "generate-css-variables --outputFileName colors _colors.scss",
```

Comment:
This has to happen in the compile step, since compiling `packages/core` depends on `@blueprintjs/colors/lib/scss/colors` existing